src/sdk/pynni/nni/gridsearch_tuner/gridsearch_tuner.py

```diff
@@ -126,12 +153,37 @@ def expand_parameters(self, para):
         return ret_para
 
     def update_search_space(self, search_space):
-        '''
-        Check if the search space is valid and expand it: support only 'choice', 'quniform', randint'
-        '''
-        self.expanded_search_space = self.json2parameter(search_space)
+        """
+        Check if the search space is valid and expand it: support only ``choice``, ``quniform``, ``randint``.
+
+        Parameters
+        ----------
+        search_space: dict
+            the format could be referred to [search space spec](https://nni.readthedocs.io/en/latest/Tutorial/SearchSpaceSpec.html).
```

Comment:
Have you checked html output? I don't know if markdown href works.

Reply:
you can click but the format is not right...